snow/init.lua

```diff
@@ -0,0 +1,22 @@
+ctf_match.register_on_build_time_start(function()
+	minetest.add_particlespawner{
+		amount = 100,
+		time = 0,
+		minpos = vector.new(ctf_map.map.pos1.x - 1, ctf_map.map.h/2 - 1, ctf_map.map.pos1.z - 1),
```

Comment:
Why are the x and z coords decremented too?

Reply:
I'm assuming without that it would set them to spawn inside the barriers at the edges of the map?
I've removed this in my upcoming update anyway since from what I've seen the map coordinates can vary (both positive and negative) and it probably wouldn't be worthwhile to add the calculations to make it independent of that

Reply:
Removed